bin/tag-generator.py

```diff
@@ -4,13 +4,15 @@
 ALPINE_VERSIONS = ['3.15', '3.16', '3.17', '3.18']
 
 LTS_VERSIONS = [ "8", "11", "17"]
+AL2023_GENERIC_VERSIONS = ["20"]
 
 def generate_tags(key, version):
     update = version.split('.')[1] if (key == '8') else version.split('.')[2]
     expanded_version = f"{key}u{update}" if (key == '8') else f"{key}.0.{update}"
 
     al2_tags = [f"{key}", f"{expanded_version}", f"{expanded_version}-al2", f"{key}-al2-full", f"{key}-al2-jdk", f"{key}-al2-generic", f"{expanded_version}-al2-generic", f"{key}-al2-generic-jdk"]
     al2023_tags = [f"{key}-al2023",  f"{expanded_version}-al2023" ,f"{key}-al2023-jdk"]
+    al2023_generic_tags = [f"{key}-al2023-generic", f"{expanded_version}-al2023-generic", f"{key}-al2023-generic-jdk"]
```

Comment:
Can this be in the `if key in AL2023_GENERIC_VERSIONS:`?

Reply:
It surely can from python perspective. However, would it easier to read with all tags list grouped together on the top (along with `al2_tags` and `al2023_tags`)?